java/registry/src/main/java/io/opensaber/registry/shard/advisory/IShardAdvisory.java

```diff
@@ -0,0 +1,10 @@
+package io.opensaber.registry.shard.advisory;
+
+import io.opensaber.registry.model.DBConnectionInfo;
+
+public interface IShardAdvisory {
```

Comment:
IShardAdvisor that has a method getShard()

Reply:
IShardAdvisor has: DBConnectionInfo getShard(String attribute)
 I hope this what you are expecting?